xml/obs_best_practice_staging_workflow.xml

```diff
@@ -36,8 +36,8 @@
   <sect2>
     <title>Start using it</title>
     <para>
-      In this view, we could find all the staging projects with a request
-      associated and also if the requests have some problems.
+      In this view, we can find all the staging projects with a request
+      associated and the current state of them.
```

Comment:
with an associated request and their current state.

Reply:
@dmarcoux changed it :)